src/applause-button.js

```diff
@@ -52,37 +58,39 @@ class ApplauseButton extends HTMLElement {
+            </g>
+          </svg>
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="-10 -10 20 20" aria-hidden="true">
+            <g class="sparkle">
+            ${arrayOfSize(SPARKLE_COUNT)
```

Comment:
I wonder if it's worth a comment here to help inform what this bit of code is actually doing?

Reply:
Certainly. Now, how to describe it.... 🤔

Reply:
Take a look now....